libs/localization/src/lib/en.json

```diff
@@ -1374,12 +1374,19 @@
       "label": "Label",
       "colon": ": ",
       "startingPosition": "STARTING POSITION",
-      "endingPosition": "ENDING POSITION"
+      "endingPosition": "ENDING POSITION",
+      "predictedAnswer": "Predicted answer: ",
+      "trueAnswer": "True Answer: ",
```

Comment:
nit: should this be sentence case only, like "True answer"? or should the one above have first letter caps for all, like "Predicted Answer"? I imagine they need to be consistent/have caps on the same letters?

Reply:
Good catch!